src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -158,10 +159,16 @@ private ColumnValues getFieldValuesFromEntityType(UUID entityTypeId, String fiel
   }
 
   private static ColumnValues getCurrencyValues() {
+    List<String> excludedCurrencyCodes = List.of(
+      "XUA", "AYM", "AFA", "ADP", "ATS", "AZM", "BYB", "BYR", "BEF", "BOV", "BGL", "CLF", "COU", "CUC", "CYP", "NLG", "EEK", "XBA", "XBB",
+      "XBC", "XBD", "FIM", "FRF", "XFO", "XFU", "GHC", "DEM", "XAU", "GRD", "GWP", "IEP", "ITL", "LVL", "LTL", "LUF", "MGF", "MTL", "MRO", "MXV",
+      "MZM", "XPD", "PHP", "XPT", "PTE", "ROL", "RUR", "CSD", "SLE", "SLL", "XAG", "SKK", "SIT", "ESP", "XDR", "XSU", "SDD", "SRG", "STD", "XTS",
+      "TPE", "TRL", "TMM", "USN", "USS", "XXX", "UYI", "VEB", "VEF", "VED", "CHE", "CHW", "YUM", "ZWN", "ZMK", "ZWD", "ZWR");
```

Comment:
Maybe this could be a class constant instead of being included in the method itself?

Reply:
oh shoot good catch